### PR TITLE
clarify gazebo install on osx

### PR DESCRIPTION
**Pull request by mrpollo**
and some missing python dependencies

### CODE REVIEW

**Comment by mrpollo** [comment]
@LorenzMeier I'm running Gazebo 7 on OSX 10.11 per a note on the docs, but the wording is confusing it states we should install `gazebo6`, whats the actual limitation here?, can we just run latest gazebo?, I'll modify my PR if we can run latest or 7

File: book/simulation-gazebo.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ Mac OS requires Gazebo 6.
 
 ```sh
 # dependencies were already installed during the initial install
-brew install opencv gazebo6
+brew install homebrew/science/opencv
```
Comment (LorenzMeier) [inline book/simulation-gazebo.md]: Did you forget the brew tap step in the Mac OS install instructions? We probably should remind people about the tap here.

Comment (mrpollo) [inline book/simulation-gazebo.md]: I left that out, I wasn't sure we wanted to recommend users to do the tap, instead they can install as the full path, whatever you prefer is fine with me

File: book/simulation-gazebo.md [hunk 2]
```diff
@@ -82,6 +83,9 @@ make posix_sitl_default gazebo_standard_vtol
 cd ~/src/Firmware
 make posix_sitl_default gazebo_tailsitter
 ```
+<aside class="tip">
+On a clean environment you might need to install some python packages like `empy` or `catking_pkg` before running `make_posix_sitl_default`
```
Comment (LorenzMeier) [inline book/simulation-gazebo.md]: Again, the missing bit is following the Mac OS instructions in the initial install. I think we should fix those.

**Comment by LorenzMeier** [comment]
We're currently focusing on Gazebo 6 because its the current default most people use.

File: book/simulation-gazebo.md (hunk 2 above)
Comment (mrpollo) [inline book/simulation-gazebo.md]: I skipped those and went straight to the gazebo part, I'm going to modify this PR to include a note about the required missing step here with a link to it

**Comment by mrpollo** [comment]
Gotcha, I'll reword this PR

**Comment by LorenzMeier** [comment]
I fixed this page to include opencv:
https://github.com/PX4/Devguide/blob/master/book/starting-installing-mac.md

So now only gazebo is needed and the reader should be reminded that he needs to install the prerequisites.

**Comment by mrpollo** [comment]
I think this 4f00785 is clearer, let me know if you want me to squash the commits before proceeding with merge

**Comment by LorenzMeier** [comment]
Thanks, merged!

**Pull request merged**